scripts/data.js

```diff
@@ -8,6 +8,14 @@ module.exports = {
     deploynow:
       "https://www.gatsbyjs.com/dashboard/deploynow?url=https://github.com/gatsbyjs/gatsby-starter-contentful-homepage",
   },
+  "contentful-ts": {
+    repo: "https://github.com/gatsbyjs/gatsby-starter-contentful-homepage-ts",
+    cms: "Contentful",
+    name: "contentful",
+    dirname: "contentful-plugin",
+    demoURL: "",
+    deploynow: "",
```

Comment:
I think these can all be filled in now that the repos are up
```suggestion
    deploynow: "https://www.gatsbyjs.com/dashboard/deploynow?url=https://github.com/gatsbyjs/gatsby-starter-contentful-homepage-ts",
```